src/main/java/org/elasticsearch/index/mapper/object/DynamicTemplate.java

```diff
@@ -200,6 +204,11 @@ private List processList(List list, String name, String dynamicType) {
         return processedList;
     }
 
+
+    public boolean isEmpty() {
+        return empty;
```

Comment:
I guess we don't really need this method, and consumers could just call conf().isEmpty()?